src/arch/Sound/DSoundHelpers.cpp

```diff
@@ -191,11 +191,6 @@ RString DSoundBuf::Init( DSound &ds, DSoundBuf::hw hardware,
 	m_bPlaying = false;
 	ZERO( m_iLastCursors );
 
-	/* The size of the actual DSound buffer.  This can be large; we generally
-	 * won't fill it completely. */
-	m_iBufferSize = 1024*64;
```

Comment:
Wait this is not unused.

It's used by buffersize_frames() which is defined in the header and used by GetPosition